sklearn/preprocessing/_polynomial.py

```diff
@@ -113,6 +114,22 @@ def _combinations(n_features, degree, interaction_only, include_bias):
         return chain.from_iterable(comb(range(n_features), i)
                                    for i in range(start, degree + 1))
 
+    @staticmethod
+    def _num_combinations(n_features, degree, interaction_only, include_bias):
```

Comment:
Could you add a docstring? Maybe mention that it is equivalent to `sum(1 for _ in self._combinations(...))`.

Reply:
done.